src/function/scalar/nested/map_extract.cpp

```diff
@@ -32,10 +33,15 @@ static void MapExtractFunction(DataChunk &args, ExpressionState &state, Vector &
 	if (children[0].second->GetType().child_types()[0].second != LogicalTypeId::SQLNULL) {
 		key_value = key_value.CastAs(children[0].second->GetType().child_types()[0].second);
 	}
-	auto offsets = ListVector::Search(*children[0].second, key_value);
+	for (idx_t row = 0; row < args.size(); row++) {
```

Comment:
Same as with the cardinality function this loop will break with constant vectors with size > 1.